### ISSUE TITLE
Incorrect zhuyin, correct pinyin

### ISSUE THREAD

**Issue by onaphta** [luoliyan/chinese-support-redux#89]
with 0.11.5 I just discovered a new problem with the zhuyin transcription, the tones of some characters are wrong, while they are correct in the pinyin field. 
![imagen](https://user-images.githubusercontent.com/46018074/63068018-2de25600-bf44-11e9-9bd4-0785c42e41c6.png)

**Comment by onaphta** [comment]
I just noticed this happens with some words, but not with others.
For instance 想法 is filled as ㄒㄧㄤˊㄈㄚ˙ (xiángfa) while it should be ㄒㄧㄤˇㄈㄚˇ 
Or 那個 in the image above

**Comment by hans** [comment]
Hi, I've also noticed incorrect Zhuyin issues. For example, 总 is transcribed as ㄔㄥˊ (wrong tone and wrong segment, should be ㄗㄨㄥˇ). The pinyin transcription looks fine. See the issue in the screenshot below for a card 总是:
![Screenshot from 2020-01-26 22-26-41](https://user-images.githubusercontent.com/6968/73148551-fb0fc280-408a-11ea-932d-3ce37a3b45ed.png)

**Comment by hans** [comment]
Oh, I've discovered the problem — it's a consequence of me using simplified characters here. The [`behavior.py` code assumes that Bopomofo lookups will happen with traditional characters only](https://github.com/luoliyan/chinese-support-redux/blob/master/chinese/behavior.py#L113) -- maybe that's a fine assumption except for strange users like me who are using Zhuyin while learning simplified characters. :)

**Comment by luoliyan** [comment]
You're correct that it's a reasonable assumption, but having it fail even 10% of the time is not ideal.

After the more serious issues are addressed, this will be fixed.
